resource-aggregate/pb/events.proto

```diff
@@ -113,4 +113,12 @@ message ResourceStateSnapshotTaken {
     repeated ResourceDeletePending resource_delete_pendings = 6;
     AuditContext audit_context = 100;
     EventMetadata event_metadata = 101;
+}
+
+message DeviceCloudStatusSnapshotTaken {
+    string device_id = 1;
+    OnlineState online = 2;
```

Comment:
Status status = 2;

Reply:
We have `message Status` so it will be `message OnlineStatus`